### DISCUSSION TITLE
Bhom installation failed for excel

### DISCUSSION THREAD

**Discussion by tonglu20** [BHoM/BHoM#1426] [Q&A]
I have reinstalled Bhom for several times, but I cannot get Bhom installed in my excel. After installation,'The file format and extension of bhom-addin. XLL do not match. The file may be corrupted or unsafe. Do not open it unless you trust its source' appears every time I open excel.  I closed excel when I installed it, do you know what might cause this problem?

**Comment by FraserGreenroyd** [comment]
@tonglu20 can you confirm which version of BHoM you're trying to install?

**Comment by daniel-locatelli** [comment]
This happened to me as well on BHoM v8.0.beta.0 in Excel 16.
For some reason it installed the 32 bits add-in.
<img src="https://github.com/user-attachments/assets/1e88dbc0-be5e-44f4-994c-28e6bf64762a" width="500">

In case anyone comes across this issue, here's how to fix it:
Go to Options > Add-ins.
Manage: `Excel Add-ins` Go...
<img src="https://github.com/user-attachments/assets/205589b9-4877-4de2-9435-2857681f5e44" width="600">

Browse...
<img src="https://github.com/user-attachments/assets/2825c8e7-8ce7-4a53-baf1-eaa73ee94456" width="300">


Go to C:\ProgramData\BHoM\Assemblies - or wherever you saved BHoM
Select BHoM-AddIn64.xll
OK
<img src="https://github.com/user-attachments/assets/9232cd28-9381-480b-a8fd-6093d6ee8730" width="500">

Uncheck Bhom-Addin
Hit OK
Close and Open Excel
<img src="https://github.com/user-attachments/assets/06523218-1ac7-4381-a499-ce1d4a1b159c" width="300">
